.circleci/config.yml

```
@@ -57,6 +70,21 @@ jobs:
           paths:
             - docs/docs/_build/html/
 workflows:
+  # default trigger on push, don't double up with actions
   build-docs:
+    when:
+      not: << pipeline.parameters.GHA_Meta >>
     jobs:
       - build-docs
+  # triggered by comment to PR with make target
+  triggered-by-issue-comment:
```

```suggestion
  triggered-by-pr-comment:
```

This is probably better renamed to PR.